Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/InterpolateBackground.py

```diff
@@ -74,8 +74,9 @@ def PyExec(self):
         lowWS = mtd[self.low_ws]
         highWS = mtd[self.high_ws]
 
-        scalar = (self.interpo_temp - self.low_temp) / (self.high_temp - self.interpo_temp)
-        outputWS = lowWS + scalar * highWS - scalar * lowWS
```

Comment:
Could be simplified to  `outputWS = lowWS + scalar * highWS +(1- scalar) * lowWS`

Reply:
This is addressed with a new commit. Thank you!